### PR TITLE
fixed form submit

### PR DESCRIPTION
**Pull request by DuGBB**

### CODE REVIEW

File: assets/js/script.js [hunk 1]
```diff
@@ -1,57 +1,47 @@
 var global = {};
 var formHandlerEl = function (event) {
+    console.log("hello");
     event.preventDefault();
     var cityStateInputEl = document.getElementById("location-input").value;
     // run api call next
 
     var parksApiUrl = "https://developer.nps.gov/api/v1/parks?limit=50&start=0&q=" + cityStateInputEl + "&api_key=xG0N7G0NIR00rad6uzGstrePJgkPJ12OyeOMTr9q";
-    var foodApiUrl = "https://api.documenu.com/v2/restaurants/search/fields?state=" + cityStateInputEl + "&key=0d461c352166be6cd4a1a1e0925996b4";
```
Comment (DuGBB) [inline assets/js/script.js:8]: @LennonSimpson Why did you delete this?

File: assets/js/script.js [hunk 2]
```diff
@@ -1,57 +1,47 @@
...
+                    parksDescrip.innerText = descrip;
+                    parkNames.appendChild(parksDescrip);
                 
 
                 }
             })
         }
     });
-
-    fetch(foodApiUrl).then(function(response) {
```
Comment (DuGBB) [inline assets/js/script.js:46]: @LennonSimpson Why did you delete this?  Did you mean to put it back and forget?

**Review by thomps0189 — APPROVED** [review]
